daprdocs/content/en/developing-applications/building-blocks/cryptography/cryptography-overview.md

```diff
@@ -0,0 +1,54 @@
+---
+type: docs
+title: Cryptography overview
+linkTitle: Overview
+weight: 1000
+description: "Overview of Dapr Cryptography"
+---
+
+With the cryptography API, you can leverage cryptography in a safe and consistent way. Dapr exposes an API that allows you to ask Dapr to perform operations, such as encrypting and decrypting messages.
+
+## Why Cryptography?
+
+Applications make extensive use of cryptography, which, when implemented correctly, can make solutions safer even when data is compromised. In certain cases, you may be required to use cryptography to comply with industry regulations (banking) or legal requirements (GDPR).  
+
+However, leveraging cryptography is difficult; you need to:
+- Pick the right algorithms and options
+- Learn the proper way to manage and protect keys
+- Navigate operational complexities when you wants limit access to cryptographic key material
+
+Organizations have increasingly used tools and services to perform cryptography outside of applications, including:
+- Hashicorp Vault, Azure Key Vault, AWS Key Management Service (KMS), Google Cloud KMS, etc.  
+- On-prem Hardware Security Module (HSM) products like Thales Luna  
+
+While those products/services perform the same or very similar operations, their APIs are different.
+
+## Cryptography in Dapr
+
+Similar to how Dapr offers an abstraction on top of secret stores, Dapr offers an abstraction layer on top of key management services or vaults (for the rest of this document referred to as "vaults").
+
+With this abstraction layer, you can perform cryptographic operations without having to access raw key material. Dapr provides a selection of correctly configured algorithms that forbid the usage of unsafe algorithms and operations. 
```

Comment:
```suggestion
With Dapr, you can perform cryptographic operations without exposing cryptographic keys to your application.

- Similar to how Dapr offers an abstraction on top of secret stores, Dapr offers an abstraction layer on top of key management services or vaults (for the rest of this document referred to as "vaults").
- Additionally, Dapr includes a set of components ("built-in cryptography" components) that perform cryptographic operations within the Dapr sidecar, and which can be used when key vaults are not available. With these components, cryptographic operations are performed within Dapr's own cryptographic engine, again without exposing keys to your application.

Both kinds of components, either those leveraging key vaults or using the built-in cryptopgrahic engine in Dapr, offer the same abstraction layer, so your solution can switch between various vaults and/or built-in cryptography components as needed: for example, you can use a locally-stored key during development, and a cloud vault in production.
```

(I've removed "Dapr provides a selection of correctly-configured algorithms that forbid the usage of unsafe algorithms and operations." because it's in the bullet points below too)